Lombiq.HelpfulExtensions/Assets/Scripts/target-blank.js

```diff
@@ -9,7 +9,8 @@
                 // In the current script, we are not using them.
                 // eslint-disable-next-line no-script-url
                 !links[i].href.startsWith('javascript:') &&
-                !links[i].href.startsWith('mailto:')) {
+                !links[i].href.startsWith('mailto:') &&
```

Comment:
Is target-blank applied to local anchor links too? If yes, we should also exclude those (= anything that starts with `#`).

Reply:
No because of `links[i].hostname !== currentHostname`.